src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/ConfirmEmployer.cshtml

```diff
@@ -0,0 +1,69 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.ConfirmEmployerViewModel
+
+@{
+    ViewBag.Title = "Confirm Employer";
+}
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper" role="main">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <partial name="_ValidationSummary" />
+                <h1 class="govuk-heading-xl">Confirm employer</h1>
+                <h2 class="govuk-heading-l govuk-!-margin-bottom-0">@Model.EmployerAccountLegalEntityName</h2>
+                
+                <dl class="das-definition-list das-definition-list--inline das-definition-list--muted govuk-!-font-size-20">
+                    @*Account name:@Model.EmployerAccountName &nbsp; &nbsp; Agreement ID:@Model.EmployerAccountLegalEntityPublicHashedId*@
+                    <dt class="das-definition-list__title">Account Name</dt>
+                    <dd class="das-definition-list__definition">@Model.EmployerAccountName</dd>
+                    <dt class="das-definition-list__title">Agreement ID</dt>
+                    <dd class="das-definition-list__definition">@Model.EmployerAccountLegalEntityPublicHashedId</dd>
+                </dl>
+
+                <form method="post">
+                    <fieldset class="govuk-fieldset">
+                        @Html.AntiForgeryToken()
+                        @Html.HiddenFor(x => x.EmployerAccountLegalEntityPublicHashedId)
+                        @Html.HiddenFor(x => x.ProviderId)
```

Comment:
this always confuses me. The providerId is in the URL. Does putting it in the body give it two data sources. Should we be using the providerId from the URL